tests/FileReaderTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests;
+
+use org\bovigo\vfs\vfsStream;
+use PhpCsFixer\FileReader;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\FileReader
+ */
+final class FileReaderTest extends TestCase
+{
+    public static function tearDownAfterClass()
```

Comment:
think this should be on tearDown in case someone adds more tests after `testReadStdinCaches`

Reply:
well, `testA` creates file, then we want to drop file after `testB`.
`testC`, if it would be ever crated, has nothing to do with that file, and moving cleaning up to `tearDown` would mean it's executed after each tests, so either wasting CPU or we need more fancy logic to determine when it shall do the cleanup and when it shall not

Reply:
i'm fine with `tearDownAfterClass` solution, let us just document it (as we just aligned) ;)